tests/phpunit/CRM/RcBase/Test/UtilsBasicTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class ReflectionTestClass
+{
+    private $restricted;
+
+    public function __construct()
+    {
+        $this->restricted = 'private';
+    }
+
+    protected function privateMethod(string $param = '')
+    {
+        return "restricted_${param}";
+    }
+}
+
+/**
+ * Test Utils Basic class
+ *
+ * @group unit
+ */
+class CRM_RcBase_Test_UtilsBasicTest extends TestCase
+{
+    public function testCallingRestrictedMethodThrowsException()
```

Comment:
I don't get this test. Why is necessary? What is tested here? Same question for the next test.

Reply:
Not really necessary. I was just curious what happens, but yes there's no need to be in the final version.
They should have stayed only a one-off test for me...